index.html

```diff
@@ -0,0 +1,96 @@
+			<p class="text-center"><img class="img-fluid" src="img/logo.png"/ alt="website logo"></p>
+			<p class="text-center"><img class="img-fluid" src="img/mint.png"/ alt="mint logo"><img class="img-fluid" src="img/quickbooks.png" alt="quickbooks logo" /><img class="img-fluid" src="img/turbotax.png" alt="turbotax logo" /></p>
+
+			</div>
+		</div>
+
+		<div class="row">
+			<div class="col">
+			<div class="white-box">
+              <div class="formtitlearea">
+				<h4 class="text-center">Create an Intuit account</h4>
+				<p class="text-center formsubtitle">One account for everything Intuit, including Mint. <a href="#">Learn more</a></p>
+				</div>
+        <div class="formarea">
+          <form action="myform.cgi">
+          	<div class="form-group">
+			    <label for="emailaddress">Email address</label>
+			    <input type="email" class="form-control" id="emailaddress" name="email">
+			  </div>
+
+          
+			<div class="form-group">
+			<label for="phone">Phone (Recommended):</label>
+			<input type="tel" id="phone" class="form-control" name="phone"
+			       pattern="[0-9]{3}-[0-9]{3}-[0-9]{4}"
+			       required>
+			       <small>Standardcalls,messaging or data rates may apply</small>
+			 </div>
+            
+              <div class="form-group">
+			  Password:
+			  <input type="password" name="psw" class="form-control">
+
+			   <button type="submit" class="btn btn-blue submitBtn my-3"><i class="fa fa-lock"></i> Create Account </button>
+			  </div>
+
+			  <p class="smallFont"><em>By clicking Create account, you agree to our <a href="#">Terms</a> and have read and acknowledge our Privacy Statement.</em></p>
+
+			</form>
+
+
+		 </div>
+
+
+				<div class="tocarea smallFont">
+					Invisible reCAPTCHA by <a href="#">Google Privacy Policy</a> and Terms of Use.
+				</div>
+			</div>
+           </div>
+		</div>
+
+		<div class="row">
+			<div class="col footer">
+				<p class="smallFont text-center">Legal Privacy Security<br>
+					© 2019 Intuit, Inc. All rights reserved. Intuit, QuickBooks, QB, TurboTax, ProConnect and Mint are registered trademarks of Intuit Inc.
+					Terms and conditions, features, support, pricing, and service options subject to change without notice.</p>
+			</div>
+		</div>
+	</div>
+
+<!-- Optional JavaScript -->
+    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+
+
+  </body>
+  </html>
```

Comment:
`SUGGESTION`

Please, fix the indentation. :blush:
A good indentation is fundamental for good readability of the code.
Many IDE/code Editor  can help you with this task.
This is VS Code, for example as shown below using [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode):

![Screenshot of how to format code using VS Code](https://res.cloudinary.com/bolaah/image/upload/v1563045186/github-microverse-project/code-reviews/code-review002.gif)